doc/getting_started/installation_win.rst

```diff
@@ -85,7 +85,12 @@ packages from their respective websites.
 #. Install the required Python modules::
 
       cd %userprofile%\zephyr
-      pip3 install --user -r scripts/requirements.txt
+      pip3 install --user -r scripts/requirements.txt windows-curses
```

Comment:
Looks like it's possible: https://stackoverflow.com/a/11704396/4577158

I'll change it.

Reply:
Great, this is the better solution IMHO.

Reply:
Yeah, this is cool

Reply:
So what's the conclusion here? I vote for having scripts/requirements-windows.txt` instead of having to type `windows-curses` but I don't mind if we keep `pip3 install --user -r scripts/requirements.txt windows-curses`

Reply:
+1, though I think it should be `windows-requirements.txt` for parallelism with the existing `py2-requirements.txt` (i.e. keep the pattern that our requirements.txt files always end in `requirements.txt`, instead of being `requirements-xxx.txt` sometimes but not always).